data-service/src/main/resources/values.yaml

```diff
@@ -38,6 +38,6 @@ autoscaling:
   minReplicas: 1
   maxReplicas: 3
 
-nodeSelector: {}
-tolerations: []
-affinity: {}
+nodeSelector: { }
+tolerations: [ ]
+affinity: { }
```

Comment:
I would have thought this was better without the space between the brackets/braces